clients/da-vinci-client/src/main/java/com/linkedin/davinci/blob/BlobFinder.java

```diff
@@ -0,0 +1,9 @@
+package com.linkedin.davinci.blob;
+
+public interface BlobFinder {
+
+  // This method will look through the partitions for the store and version provided until it finds the partition
+  // requested, it will then return the URL of the instances that are ready to serve in the partition.
```

Comment:
```suggestion
  /**
   * This method will look through the partitions for the store and version provided until it finds the partition
   * requested, it will then return the URL of the instances that are ready to serve in the partition.
   */
```
I think it would be better to use docstring comments for classes and methods.